conf/ui.yaml.template

```diff
@@ -0,0 +1,36 @@
+UI:
+  SCREENSHOTS_PATH: /tmp/robottelo/screenshots/
+  # Webdriver logging options
+  LOG_DRIVER_COMMANDS:
+  - newSession
+  - windowMaximize
+  - get
+  - findElement
+  - sendKeysToElement
+  - clickElement
+  - mouseMoveTo
+
+  # browser tells robottelo which browser to use when testing UI. Valid values (selenium/remote)
+  BROWSER: selenium
+  # Webdriver to use
+  WEBDRIVER: chrome
+  WEBDRIVER_BINARY: /usr/bin/chromedriver
+
+  # web_kaifuku settings
+  WEBKAIFUKU:
+    webdriver: selenium
+    webdriver_options:
+      command_executor: http://localhost:4444/wd/hub
+      desired_capabilities:
+        browserName: chrome
+        chromeOptions:
```

Comment:
Few questions and optional things:
- why repeating `webdriver:` in ui and webkaifuku blocks? isn't the same?
- we should remove these UI settings from  `robottelo.yaml.template` and add some help texts here?
- we should keep these settings consistent with others in capital letters, just a thought.

Reply:
> why repeating webdriver: in ui and webkaifuku blocks? isn't the same?

Please look at the CI PR 399 this will clear you doubt, Webkaifuku select either webdriver name (chrome, firefox ) or the remote. The references are copied.

>   we should remove these UI settings from robottelo.yaml.template and add some help texts here?

I will update that and remove those code. I guess email would be nice rather having that help text as we are moving everything in ui.yaml

> we should keep these settings consistent with others in capital letters, just a thought.

I even wanted in the same manner seems like we need pass the same object in the airgun first and then it is passed the webkaifuku. Current yaml configuration do not required to write any extra layer. Same configuration is supported in webkaifuku hence I avoided (explicit good than implicit )

Reply:
@latran some of these topics are being discussed in the airgun PR introducing kaifuku, as its most relevant there where the direct dependency lies.